addons/account_peppol/models/res_company.py

```diff
@@ -192,6 +193,12 @@ def _compute_account_peppol_phone_number(self):
                 except ValidationError:
                     continue
 
+    @api.depends('account_peppol_proxy_state')
+    def _compute_peppol_can_send(self):
+        for company in self:
+            can_send_domain = self.env['account_edi_proxy_client.user']._get_can_send_domain()
```

Comment:
Can be moved outside of the loop I guess

Reply:
obviously, my bad